### ISSUE TITLE
Czech localization

### ISSUE THREAD

**Issue by blahetal** [sveltia/sveltia-cms#266]
I can help with this

**Comment by cebreus** [comment]
I can help with this, too.

**Comment by DucLongHoang** [comment]
sign me up too

**Comment by dominikfryc** [comment]
+1

**Comment by kyoshino** [comment]
Sorry for keeping you waiting for so long! Sveltia CMS is now ready for localization. We’ll be adding more strings before and after the 1.0 release, but it’s a good time to start translating. Check out the [README](https://github.com/sveltia/sveltia-cms/blob/main/src/lib/locales/README.md) for instructions. Let me know if you have any questions!

**Comment by nula1337** [comment]
Hello, I have the translations ready, I'm not sure if @blahetal is active here. @kyoshino can you assign me to this issue, please?

**Comment by kyoshino** [comment]
No. If @blahetal is inactive, I’ll assign the issue to @cebreus, the second person on board.

**Comment by blahetal** [comment]
> No. If [@blahetal](https://github.com/blahetal) is inactive, I’ll assign the issue to [@cebreus](https://github.com/cebreus), the second person on board.

Hello, currently I am pressed for time, can you assign the issue to the next person in line.

**Comment by nula1337** [comment]
> No. If [@blahetal](https://github.com/blahetal) is inactive, I’ll assign the issue to [@cebreus](https://github.com/cebreus), the second person on board.

Sure thing, here are the translations for anyone interested, you can use them as baseline:
- **Sveltia UI**: https://pastebin.com/Nj0iZYs9
- **Sveltia CMS**: https://pastebin.com/qRQSZCEf

**Comment by kyoshino** [comment]
@cebreus Do you have time to work on this?

**Comment by kyoshino** [comment]
@DucLongHoang Can you still work on this?

**Comment by kyoshino** [comment]
@dominikfryc Can you work on this now?

**Comment by dominikfryc** [comment]
@kyoshino Yes, I'll do it.

**Comment by kyoshino** [comment]
Czech shipped with [v0.188.0](https://github.com/sveltia/sveltia-cms/releases/tag/v0.188.0) 🇨🇿

**Comment by kyoshino** [comment]
@dominikfryc Hi there! I just synced the latest English strings. To save you time, I had Claude translate the new ones for you. Please review the translations and update them if necessary. [Here’s the diff](https://github.com/sveltia/sveltia-cms/commit/a0a0851d05bf79298e5ba824a14b6bad7708f871#diff-ebea13d6ef6a21bf593ade6c9a623ccdf1edf4c7ba28da37b4af82a7ba5b405e).

Separately, Claude flagged some pluralization issues in the translations. Please [check those](https://github.com/sveltia/sveltia-cms/commit/35e6c85ba2f3fc9734b9c745da782f1389f00f1f#diff-ebea13d6ef6a21bf593ade6c9a623ccdf1edf4c7ba28da37b4af82a7ba5b405e) as well to ensure they are accurate and contextually appropriate. If you have any questions or need further assistance, feel free to reach out. Thank you for your help!

**Comment by dominikfryc** [comment]
@kyoshino Hi! All translations seem to be correct, good job!
